### DISCUSSION TITLE
Use launch menu for ssh connections.

### DISCUSSION THREAD

**Discussion by LucNig** [wezterm/wezterm#422] [Q&A] [answered]
I tried to add an ssh connection to my laucher menu. but somehow i am missing something or it does not work this way;

my config part:

**local launch_menu = {
      label = "server x",
      -- The argument array to spawn.  If omitted the default program
      -- will be used as described in the documentation above
      args = {"ssh", "root@10.10.10.10"},
    },**

**Comment by wez** [comment, marked as answer]
It's not clear from your post whether you have a full config file; you want something like this:

```lua
return {
   launch_menu = {
       {
           label = "server x",
           args = {"ssh", "root@10.10.10.10"},
       },
   },
}
```

The launch menu is itself an array of launch menu items, so pay attention to how many `{}` you have around your menu definition!

Are you experiencing an error, or is what you're trying not producing a menu item, or is the menu item there, but not doing what you expect?

**Reply by LucNig** [reply]
Hi Wez,

Thanks for your quick response! I missed the { before "label". After adding it the item works.
Now it opens in a new windows instead of a tab, but guess i can change that somewhere.

Strange thing was, it did not trew an error, just nothing happens.
